Makefile

```diff
@@ -136,7 +136,7 @@ prepare-server:
 
 only-install-server: prepare-server build-server install-server-package
 
-install-server: lint-server only-install-server # Install backend
+install-server: lint-server only-install-server ## Install backend
```

Comment:
Since the `install` target is described as "Build and install", it seems like this description should be "Build and install backend only".

This also leads to one asking whether there should be a similar _frontend-only_ target.  I.e., do we need to expose `install-ui`?

Also, in looking at the targets, `check-install` seems generally useful and could be helpful in confirming a dev-based installation.  Not sure if we should document that more formally (i.e., mention it in the docs and add it to the default `make` output).

Reply:
lol, I told @karlaspuldaro these questions were going to come :) I will let you guys decide.

Reply:
As discussed in today's meeting to gather an overall team opinion, it was decided
1. not to target `install-ui` since `watch` seems more useful for frontend dev and it's already documented
2. `check-install` is a good candidate to be exposed and documented, maybe it needs a better name, but for now we'll create a separate issue for it